src/sources/splunk_hec/mod.rs

```diff
@@ -980,21 +989,22 @@ fn parse_timestamp(t: i64) -> Option<DateTime<Utc>> {
     Some(ts)
 }
 
-/// Maintains last known extracted value of field and uses it in the absence of field.
-struct DefaultExtractor {
+/// MetaExtractor is a helper struct that extracts a field from the request and adds it to the log event metadata.
+/// It maintains last known extracted value of field and uses it in the absence of field.
+struct MetaExtractor {
     field: &'static str,
     to_field: OptionalValuePath,
     value: Option<Value>,
     log_namespace: LogNamespace,
 }
 
-impl DefaultExtractor {
+impl MetaExtractor {
     const fn new(
         field: &'static str,
         to_field: OptionalValuePath,
         log_namespace: LogNamespace,
     ) -> Self {
-        DefaultExtractor {
+        MetaExtractor {
```

Comment:
nit: This and all other references within `impl MetaExtractor` could just be `Self`.